Dockerfile

```
@@ -0,0 +1,68 @@
+FROM archlinux:base-devel AS build
+
+# Setup sudo user & install dependencies
+RUN pacman -Sy --noconfirm git pacutils perl-json-xs devtools pacman-contrib ninja cargo && \
```

```suggestion
RUN pacman -Syu --noconfirm git pacutils perl-json-xs devtools pacman-contrib ninja cargo && \
```

Running the docker build currently fails, I think because of the partial upgrade here (cargo out of sync with glibc)
```
cargo: /usr/lib/libm.so.6: version `GLIBC_2.38' not found (required by cargo)
make: *** [makefile:6: all] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```